### PR TITLE
package: glew

### PR DESCRIPTION
**Pull request by mesaglio**
This package is needed to build opencv

### CODE REVIEW

**Comment by mesaglio** [comment]
@ajayk can you help me with this?

File: glew.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://downloads.sourceforge.net/glew/glew-${{package.version}}.tgz
+      expected-sha512: 57453646635609d54f62fb32a080b82b601fd471fcfd26e109f479b3fef6dfbc24b83f4ba62916d07d62cd06d1409ad7aa19bc1cd7cf3639c103c815b8be31d1
+
+  - runs: |
+      cd build
+      cmake -DBUILD_UTILS=ON -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_BUILD_TYPE=MinSizeRe ./cmake
```
Comment (ajayk) [inline glew.yaml:30]: Can we use the cmake pipeline

**Review by ajayk — CHANGES_REQUESTED** [review]

File: glew.yaml (hunk 1 above)
Comment (mesaglio) [reply glew.yaml:30]: If I add
```yaml
 - working-directory: /home/build/build/cmake
     uses: cmake/configure
     with:
       opts: |
         -DBUILD_UTILS=ON 
```

the binaries arent generated and have sobpackage `mv` error
```bash
2024/05/03 21:40:21 WARN mv: can't rename '/home/build/melange-out/glew/usr/bin/glewinfo': No such file or directory
2024/05/03 21:40:21 ERRO Step failed: exit status 1
```

Comment (mesaglio) [reply glew.yaml:31]: what do you think?

Comment (ajayk) [reply glew.yaml:31]: Let me get back to you on this I was able to build this locally with the cmake pipelines

Comment (mesaglio) [reply glew.yaml:31]: Thank you!

**Review by ajayk — APPROVED** [review]

**Pull request merged**